sdk/formrecognizer/ai-form-recognizer/review/ai-form-recognizer.api.md

```diff
@@ -0,0 +1,1299 @@
+## API Report File for "@azure/ai-form-recognizer"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+/// <reference types="node" />
+
+import { AzureKeyCredential } from '@azure/core-auth';
+import { KeyCredential } from '@azure/core-auth';
+import { OperationOptions } from '@azure/core-client';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
+import { PipelineOptions } from '@azure/core-rest-pipeline';
+import { PollerLike } from '@azure/core-lro';
+import { PollOperationState } from '@azure/core-lro';
+import { TokenCredential } from '@azure/core-auth';
+
+// @public
+export type AnalysisPoller<Result = AnalyzeResult<AnalyzedDocument>> = PollerLike<DocumentAnalysisPollOperationState<Result>, Result>;
+
+// @public
+export interface AnalyzedDocument {
+    boundingRegions?: BoundingRegion[];
+    confidence: number;
+    docType: string;
+    fields: {
+        [field: string]: DocumentField;
+    };
+    spans: DocumentSpan[];
+}
+
+// @public
+export interface AnalyzeDocumentsOptions<Result = AnalyzeResult<AnalyzedDocument>> extends OperationOptions, PollerOptions<DocumentAnalysisPollOperationState<Result>> {
+    locale?: string;
+    pages?: string[];
+}
+
+// @public
+export interface AnalyzeResult<Document = AnalyzedDocument> {
+    apiVersion: FormRecognizerApiVersion;
+    content: string;
+    documents: Document[];
+    entities: DocumentEntity[];
+    keyValuePairs: DocumentKeyValuePair[];
```

Comment:
are these pairs model-specific? if not, perhaps we can use a clearer name here.

Reply:
No, they're not model-specific. What do you mean by a clearer name, here? Type, or value?

Reply:
both. The names here describe the structure of the value but not what it really is.

Reply:
We can consider this in the future, but without a concrete alternative for the name (what the type represents is a directed association between document elements, one of which is a key and one is a value, such as a labeled field, for example), I won't rename this for beta.1 tomorrow.

Reply:
Sounds good. https://github.com/Azure/azure-sdk-for-js/pull/17955#discussion_r721642423 applies to all proposed renames and of course I will ultimately leave it to the FR crew.

Reply:
OK cool. I'm resolving all other comments related to naming for now.